playbooks/openshift/heat_deprovision.yml

```diff
@@ -12,14 +12,15 @@
         server: "{{ cluster_name }}-lb-0"
         floating_ip_address: "{{ openshift_public_ip }}"
         state: absent
-      when: lb_vm_group_size > 0
+      when: master_vm_group_size > 1
```

Comment:
The logic here is that the condition matches the condition used in provisioning. See below.